framework/web/migrations/m160313_153426_session_init.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+use yii\base\InvalidConfigException;
+use yii\web\DbSession;
+use yii\db\Migration;
+
+/**
+ * Initializes Session tables
+ *
+ * @author Misbahul D Munir <misbahuldmunir@gmail.com>
+ * @since 2.0.8
+ */
+class m160313_153426_session_init extends Migration
+{
+
+    /**
+     * @throws yii\base\InvalidConfigException
+     * @return DbSession
+     */
+    protected function getSession()
```

Comment:
> For me. I prefer create table automatically when table does not exists like in Yii1.

I'm strongly against this idea. I don't want Yii to touch my DB without my explicit command to do it.